nekoyume/Assets/_Scripts/UI/Widget/Popup/BattleResultPopup.cs

```diff
@@ -614,8 +614,16 @@ private IEnumerator CoUpdateBottom(int limitSeconds)
                                       SharedModel.StageType == StageType.EventDungeon;
             if (isActionPointEnough)
             {
-                repeatButton.gameObject.SetActive(canExit);
-                repeatButton.interactable = canExit;
+                if (SharedModel.IsClear)
+                {
+                    stagePreparationButton.gameObject.SetActive(canExit);
+                    stagePreparationButton.interactable = canExit;
+                }
+                else
+                {
+                    repeatButton.gameObject.SetActive(canExit);
+                    repeatButton.interactable = canExit;
+                }
```

Comment:
문맥을 이해한 건 아니지만, 혹시 이렇게 각 분기에 나머지 오브젝트 활성화 여부를 조정해줄 필요는 없나요?
```suggestion
                if (SharedModel.IsClear)
                {
                    repeatButton.gameObject.SetActive(false);
                    stagePreparationButton.gameObject.SetActive(canExit);
                    stagePreparationButton.interactable = canExit;
                }
                else
                {
                    stagePreparationButton.gameObject.SetActive(false);
                    repeatButton.gameObject.SetActive(canExit);
                    repeatButton.interactable = canExit;
                }
```

Reply:
네 요거 호출전에 false로 초기화해주더라구요~!

Reply:
오 넵!